### PR TITLE
Feature/ratings

### PR DESCRIPTION
**Pull request by nitheshkumar-hcl**
Committing the ui changes for rating story

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -188,5 +188,6 @@ Bundle-DocURL:
             <groupId>uk.org.lidalia</groupId>
             <artifactId>slf4j-test</artifactId>
         </dependency>
```
Comment (rakeshkumar-verma) [inline core/pom.xml:190]: @nitheshkumar-hcl  Why we need this dependency?

File: core/src/main/java/com/hcl/ecomm/core/services/impl/AddToCartServiceImpl.java [hunk 2]
```diff
@@ -165,10 +165,4 @@ public JSONObject updateCartItem(JSONObject item, String itemId, String custToke
 
 
 
-	
-
-
-
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/services/impl/AddToCartServiceImpl.java:171]: @nitheshkumar-hcl  What is changes in this file?

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/ratings/ratings.html.dir/.content.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="nt:file">
+    <jcr:content
+        jcr:lastModifiedBy="Administrator"
+        jcr:mimeType="text/plain"
+        jcr:primaryType="nt:resource"/>
+</jcr:root>
```
Comment (rakeshkumar-verma) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/ratings/ratings.html.dir/.content.xml]: @nitheshkumar-hcl  Why we need this file?

**Review by rakeshkumar-verma — CHANGES_REQUESTED** [review]
@nitheshkumar-hcl  pls incorporate review comments.

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/ratings/ratings.html.dir/.content.xml (hunk 3 above)
Comment (nitheshkumar-hcl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/ratings/ratings.html.dir/.content.xml]: Removed the file as per the review comments

File: core/src/main/java/com/hcl/ecomm/core/services/impl/AddToCartServiceImpl.java (hunk 2 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/services/impl/AddToCartServiceImpl.java:171]: Committing the changes as per review comments

File: core/pom.xml (hunk 1 above)
Comment (nitheshkumar-hcl) [reply core/pom.xml:190]: It's already existing in development branch,didn't added it as part of this PR.

File: core/src/main/java/com/hcl/ecomm/core/services/impl/AddToCartServiceImpl.java (hunk 2 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/services/impl/AddToCartServiceImpl.java:171]: Removed the spaces in the file

File: core/pom.xml (hunk 1 above)
Comment (rakeshkumar-verma) [reply core/pom.xml:190]: @nitheshkumar-hcl  can you remove this file from PR.?

**Pull request merged**